opentreemap/stormwater/migrations/0007_drainage_area_permissions.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from treemap.audit import add_default_permissions
+from treemap.instance import Instance
+
+
+def add_permissions_all(apps, schema_editor):
+    all_instances = Instance.objects.all()
+    for instance in all_instances:
+        map_feature_type_set = set(instance.map_feature_types) - {'Plot'}
+        if 0 < len(map_feature_type_set):
+            add_default_permissions(instance)
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('stormwater', '0006_stormwater_drainage_area'),
+        ('treemap', '0011_instance_universal_rev'),
+        ('treemap', '0026_add_canopy_fields'),
+    ]
+
+    operations = [
+        migrations.RunPython(add_permissions_all)
```

Comment:
You should add a `noop` backwards migration, so that we can rollback this migration if necessary (see https://docs.djangoproject.com/ja/1.9/ref/migration-operations/#django.db.migrations.operations.RunPython.noop)